openslides_backend/action/actions/user/json_upload.py

```diff
@@ -0,0 +1,174 @@
+        data = instance.pop("data")
+
+        # validate and check for duplicates
+        self.init_duplicate_set(
+            [
+                {
+                    field: entry.get(field, "")
+                    for field in ("username", "first_name", "last_name", "email")
+                }
+                for entry in data
+            ]
+        )
+        self.rows = [
+            self.generate_entry(entry, payload_index)
+            for payload_index, entry in enumerate(data)
+        ]
+
+        # generate statistics
+        itemCount = len(self.rows)
+        status_to_count = {status: 0 for status in ImportStatus}
+        for entry in self.rows:
+            status_to_count[entry["status"]] += 1
+
+        raw_statistics = (
+            ("total", itemCount),
+            ("created", status_to_count[ImportStatus.NEW]),
+            ("updated", status_to_count[ImportStatus.DONE]),
+            ("error", status_to_count[ImportStatus.ERROR]),
+            ("warning", status_to_count[ImportStatus.WARNING]),
+        )
+        self.statistics = [
+            {"name": name, "value": value} for name, value in raw_statistics
+        ]
+
+        self.set_status(
+            status_to_count[ImportStatus.ERROR], status_to_count[ImportStatus.WARNING]
+        )
+        self.store_rows_in_the_action_worker("account")
+        return {}
+
+    def generate_entry(
+        self, entry: Dict[str, Any], payload_index: int
+    ) -> Dict[str, Any]:
+        status, error = None, []
+        try:
+            UserCreate.schema_validator(entry)
+            if entry.get("username"):
+                if self.check_username_for_duplicate(entry["username"], payload_index):
+                    status = ImportStatus.DONE
+                    if searchdata := self.get_search_data(payload_index):
+                        entry["id"] = searchdata["id"]
+                    else:
+                        status = ImportStatus.ERROR
+                        error.append(f"Duplicate in csv list index: {payload_index}")
+                else:
+                    status = ImportStatus.NEW
+                entry["username"] = {"value": entry["username"], "info": "done"}
+            else:
+                if not (entry.get("first_name") or entry.get("last_name")):
+                    status = ImportStatus.ERROR
+                    error.append("Cannot generate username.")
+                elif self.check_name_and_email_for_duplicate(
+                    *UserJsonUpload._names_and_email(entry), payload_index
+                ):
+                    status = ImportStatus.DONE
+                    if searchdata := self.get_search_data(payload_index):
+                        entry["username"] = {
+                            "value": searchdata["username"],
+                            "info": ImportStatus.DONE,
+                        }
+                        entry["id"] = searchdata["id"]
```

Comment:
I thought we will store this in the "username" object. This way there is an unknown column for the client and in committee-import you'll have several fields with there user_id and have to store the id in its object. Changed it in wiki